docs/content/latest/manage/backup-restore/point-in-time-restore-ysql.md

```diff
@@ -0,0 +1,331 @@
+    ```
+
+1. Verify that the snapshot is complete:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                         State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    No snapshot restorations
+    ```
+
+### Restore from an absolute time
+
+1. Get a timestamp:
+
+    ```sql
+    yugabyte=# select extract(epoch from now())*100000 as current_timestamp;
+    ```
+
+    ```output
+    current_timestamp 
+    -------------------
+      161767067918510
+    (1 row)
+    ```
+
+1. Add a row for employee 9999 to the table:
+
+    ```sql
+    INSERT INTO employees (employee_no, name, department, salary) 
+      VALUES 
+      (9999, 'Wrong Name', 'Marketing', 10000);
+
+    SELECT * FROM employees;
+    ```
+
+    ```output
+    employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1223 | Lucille Ball   | Operations |  70000
+            9999 | Wrong Name     | Marketing  |  10000
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+    (5 rows)
+    ```
+
+1. List snapshots:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                         State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    No snapshot restorations
+    ```
+
+1. Restore the latest snapshot to the timestamp you obtained before you deleted the data:
+
+    ```sh
+    # NOTE add a zero (0) to the end of the timestamp!
+    $ bin/yb-admin restore_snapshot bb5fc435-a2b9-4f3a-a510-0bacc6aebccf 1617670679185100
+
+1. Verify the snapshot is restored:
+
+    ```sh
+    $ bin/yb-admin list_snapshots
+    ```
+
+    ```output
+    Snapshot UUID                     State
+    bb5fc435-a2b9-4f3a-a510-0bacc6aebccf  COMPLETE
+    Restoration UUID                      State
+    bd7e4e52-b763-4b95-87ce-9399e1ac206e  RESTORED
+    ```
+
+1. Verify the data is restored:
+
+    ```sql
+    yugabyte=# select * from employees;
+    ```
+
+    ```output
+    employee_no |      name      | department | salary 
+    -------------+----------------+------------+--------
+            1223 | Lucille Ball   | Operations |  70000
+            1224 | John Zimmerman | Sales      |  60000
+            1221 | John Smith     | Marketing  |  50000
+            1222 | Bette Davis    | Sales      |  55000
+    (4 rows)
+    ```
+
+### Restore from a relative time
+
+In addition to restoring to a particular timestamp, you can also restore to a relative time, such as "ten minutes ago". In this example, you'll delete some data from the existing `employees` table, then restore the state of the database to what it was five minutes prior.
+
+1. Wait five minutes after you complete the steps in the previous section. This is so that you can use a known relative time for the restore.
```

Comment:
not sure if worth mentioning instead / as well, the side effect, that if you have not waited 5 minutes, you might restore to the state of the database, before you even inserted the initial rows?

Reply:
I'll adjust pending your/Nicolas's response in slack :)

Reply:
Updated